core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -105,6 +105,10 @@
                                   style="margin-left: 10px;">
         </exploration-summary-tile>
       </div>
+
+      <div>
+        <a ng-href="/collection/{{collectionId}}" class="conversation-skin-back-to-collection" translate="I18N_PLAYER_BACK_TO_COLLECTION"></a>
```

Comment:
Yeah... Grepping would probably been a good idea. I actually stumbled upon ng ref on stack overflow when using href didn't work. Thanks for the tip